ES/ESdd017.xml

```diff
@@ -88,105 +90,129 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      <supportDesc>
                         <support>
                            <material key="parchment"/>
+                           Holes are amended on <locus target="#57 #132 #139 #155 #164"/>.
                         </support>
-                        <extent>Made of <measure unit="leaf">185.0</measure> folios, of which
-                              <measure unit="leaf" type="blank"/> blank 58vc, 184rbc, 184v, 185rv,
-                           in <measure unit="quire">23.0</measure> quires (22 + B). <dimensions type="outer">
+                        <extent><measure unit="leaf" quantity="185">185</measure>
+                           <measure unit="page" type="blank"/>5 <locus target="#58vc #184rbc #184v #185rv"/>
```

Comment:
i think here you need to close the tag after you give the locus: <measure unit="page" type="blank"><locus ....></measure>